src/main/kotlin/com/carneirofatima/vehicleterminal/ValidationRequest.kt

```diff
@@ -0,0 +1,26 @@
+package com.carneirofatima.vehicleterminal
+
+import com.carneirofatima.vehicleterminal.dao.DriverDao
+import com.carneirofatima.vehicleterminal.to.DriverTO
+import org.springframework.stereotype.Component
+
+@Component
+class ValidationRequest(
```

Comment:
I don't like to add DAO dependency to Validation class. Perhaps you can put database validation inside `service` class